.github/workflows/run-tests.yml

```diff
@@ -58,6 +58,7 @@ jobs:
 
 ################################################################
       - name: Build neofs-node (neofs-cli and neofs-adm)
+        timeout-minutes: 30
```

Comment:
I'd be a bit more aggressive here, 30m is too much to build some Go binaries.

Reply:
Done